docs/game.yml

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
I don't think we need to duplicate this file in our repo. We can probably just link to the example in our docs. https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/main/docs/examples/2048/2048_full.yaml

Reply:
That link does not use the default namespace. It uses another namespace.

Reply:
Is that the only difference?

Reply:
yes

Reply:
Eventually this will be removed when I get the cluster creation fully working.

Reply:
So, could you add that to the doc. Like
> * Create Ingress controller fargate
Download config [here](https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/main/docs/examples/2048/2048_full.yaml) and replace all instances of namespace `game-2048` with `default`.